### PR TITLE
Tools docs

### PR DESCRIPTION
**Pull request by andrewprzh**
- Add links to standalone tools
- Add references for binspreader and spaligner
- Move spaligner README to docs
- Move pathracer README to docs

### CODE REVIEW

File: docs/binspreader.md [hunk 1]
```diff
@@ -16,6 +16,16 @@ source of information for refining. Optionally, BinSPreader can be provided with
 multiple Hi-C and/or paired-end libraries. The [BinSPreader protocol](https://star-protocols.cell.com/protocols/2802) contains more detailed
 instructions on installing and running BinSPreader.
 
+## Compilation
+
+To compile SPAligner, run
```
Comment (asl) [inline docs/binspreader.md:21]:
```suggestion
To compile BinSPreader, run
```

File: docs/binspreader.md [hunk 2]
```diff
@@ -16,6 +16,16 @@ source of information for refining. Optionally, BinSPreader can be provided with
...
+
+```
+./spades_compile -SPADES_ENABLE_PROJECTS=binspreader
+```
+
+After the compilation is complete, `binspreader` executable will be located in the `bin/` folder.
```
Comment (asl) [inline docs/binspreader.md:27]:
```suggestion
After the compilation is complete, `binspreader` executable will be located in the `bin` folder.
```

File: docs/installation.md [hunk 3]
```diff
@@ -88,7 +88,7 @@ for example:
 
 which will install SPAdes into `/usr/local/bin`.
 
-After installation you will get the same files (listed above) in `./bin` directory (or `<destination_dir>/bin` if you specified PREFIX). We also suggest adding `bin` directory to the `PATH` variable.
+After installation, you will get the same files (listed above) in `./bin` directory (or `<destination_dir>/bin` if you specified PREFIX). We also suggest adding `bin` directory to the `PATH` variable.
```
Comment (asl) [inline docs/installation.md:91]:
```suggestion
After installation, you will get the same files (listed above) in `bin` directory (or `<destination_dir>/bin` if you specified PREFIX). We also suggest adding `bin` directory to the `PATH` variable.
```

File: docs/pathracer.md [hunk 4]
```diff
@@ -22,21 +19,33 @@ translation on-fly walking through frameshifts.
...
+![Scheme of extended pHMM](./pHMM_with_frameshifts.svg)
 
 but for `pathracer-seq-fs` this extension is crucial: for aligning amino-acid pHMMs without allowing indels in the nucleotide space
 six frame translation + `hmmsearch` from **HMMer** package is more than enough.
 
-### Input
-Currently the tool supports only _de Bruijn_ graphs in GFA format as produced by **SPAdes** or compatible assembler in this matter (e.g., **MEGAHIT**).
+## Compilation
+
+To compile SPAligner, run
```
Comment (asl) [inline docs/pathracer.md:29]:
```suggestion
To compile PathRacer, run
```

File: docs/pathracer.md [hunk 5]
```diff
@@ -22,21 +19,33 @@ translation on-fly walking through frameshifts.
...
+
+```
+./spades_compile -SPADES_ENABLE_PROJECTS=pathracer
+```
+
+After the compilation is complete, `pathracer` executable will be located in the `bin/` folder.
```
Comment (asl) [inline docs/pathracer.md:35]:
```suggestion
After the compilation is complete, `pathracer` executable will be located in the `bin` folder.
```

File: docs/spaligner.md [hunk 6]
```diff
@@ -1,31 +1,57 @@
...
+
+## Compilation
+
+To compile SPAligner, run
+
+```
+./spades_compile -SPADES_ENABLE_PROJECTS=spaligner
+```
+
+After the compilation is complete, `spaligner` executable will be located in the `bin/` folder.
```
Comment (asl) [inline docs/spaligner.md:16]:
```suggestion
After the compilation is complete, `spaligner` executable will be located in the `bin` folder.
```

**Comment by andrewprzh** [comment]
Fixes #1302

**Pull request merged**